lib/loader/holding_loader.rb

```diff
@@ -27,12 +27,6 @@ def item_from_line(_line)
     def load(batch)
       Clustering::ClusterHolding.new(batch).cluster
     end
-
-    def final_line
```

Comment:
AFAICT this was never getting called (there was no way for @update to be set). In any case it wouldn't have worked with the chunking, and regardless we need to have another method of indicating when all chunks are done and then doing the deletes as a separate job.

Reply:
Yeah I think this was ever only aspirational.